service/node.go

```diff
@@ -37,16 +37,16 @@ import (
 
 func (s *service) NodeExpandVolume(
 	context.Context,
-	*csi.NodeExpandVolumeRequest) (*csi.NodeExpandVolumeResponse, error) {
-
+	*csi.NodeExpandVolumeRequest,
```

Comment:
If we remove that comma, then it will break the build - the compiler requires commas if the expression is expected to continue. This is required and comes directly from the language specification. It's one of the basic features of Go languages, please follow the following examples and video to learn more about Go compiler.

- [inline, run successful](https://go.dev/play/p/thHkUScqeVs)
- [comma added, compile error](https://go.dev/play/p/_L70HscIduD)
- [no comma, compile error](https://go.dev/play/p/Sz9POW4la-U)

https://youtu.be/c_cdmmTmS1U?si=1JusTidF-oWHwSX3&t=548

Generally, the good practice in Go is to split all arguments into separate lines when not putting them in line. This was part of the automatic fix by linters (please start linting your code to avoid issues like this in future @shefali-malhotra ).

Reply:
Understood . Thanks for sharing the details.